### DISCUSSION TITLE
Possible to alter user-agent from Liquidsoap to custom string?

### DISCUSSION THREAD

**Discussion by gavin2812** [savonet/liquidsoap#3006] [User Support]
Hello,

I am using this as my input to stream onto another server:

```
url = "http://incomingstream.com/stream"
input = mksafe(input.http(url))
```

On the server I'm connecting the input to, it  appears as `User-Agent: Liquidsoap/2.1.4 (Unix; OCaml 4.14.0)',`

My question is, is it possible to alter the user-agent to a custom string? If yes, how could this be done to specify it in the input section?

Thanks in advance.

**Comment by smimram** [comment]
Yes, it should be something like
```
input = mksafe(input.http(user_agent="my agent", url))
```

**Comment by gavin2812** [comment]
@smimram Awesome! Many thanks for quick reply, works perfectly!
